### PR TITLE
PROD-1403: Omit specific fields from system payload if empty

### PR DESCRIPTION
**Pull request by TheAndrewJackson**
Closes PROD-1403

### Description Of Changes

This PR aligns the FE with how the bulk vendor add route handles these fields.

### Code Changes

* [ ] Omit `administrating_department` and `joint_controller_info` if they're empty

### Steps to Confirm

* [ ] submit a form with them empty and check the network tab to make sure they're excluded

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* Documentation:
  * [ ] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
* [x] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [x] Update `CHANGELOG.md`
* [ ] For API changes, the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated

### CODE REVIEW

File: clients/admin-ui/src/features/system/form.test.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      uses_non_cookie_access: false,
+      legitimate_interest_disclosure_url: "",
+    };
+
+    const payload = transformFormValuesToSystem(formValues);
+    expect(payload).not.toHaveProperty("administrating_department");
+    expect(payload).not.toHaveProperty("joint_controller_info");
+  });
+
+  it("should keep joint controller into and adminitstrating department if not empty", () => {
```
Comment (allisonking) [inline clients/admin-ui/src/features/system/form.test.ts:41]:
```suggestion
  it("should keep joint controller into and administrating department if not empty", () => {
```

File: clients/admin-ui/src/features/system/form.test.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import { describe, expect, it } from "@jest/globals";
+
+import { transformFormValuesToSystem } from "~/features/system/form";
+
+describe("transformFormValuesToSystem", () => {
+  it("should omit joint controller into and adminitstrating department if empty", () => {
```
Comment (allisonking) [inline clients/admin-ui/src/features/system/form.test.ts:6]:
```suggestion
  it("should omit joint controller into and administrating department if empty", () => {
```

File: clients/admin-ui/src/features/system/form.test.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import { describe, expect, it } from "@jest/globals";
+
+import { transformFormValuesToSystem } from "~/features/system/form";
+
+describe("transformFormValuesToSystem", () => {
```
Comment (allisonking) [inline clients/admin-ui/src/features/system/form.test.ts:5]: woot thanks for adding this unit test!

**Review by allisonking — APPROVED** [review]
nice!

File: clients/admin-ui/src/features/system/form.test.ts (hunk 3 above)
Comment (TheAndrewJackson) [reply clients/admin-ui/src/features/system/form.test.ts:5]: you're welcome!

**Pull request merged**